aptos-move/move-examples/aggregator_examples/Move.toml

```diff
@@ -6,5 +6,8 @@ version = "0.0.0"
 aggregator_examples = "0xCAFE"
 
 [dependencies]
-AptosStdlib = { git = "https://github.com/aptos-labs/aptos-framework.git", subdir = "aptos-stdlib", rev = "mainnet" }
-AptosFramework = { git = "https://github.com/aptos-labs/aptos-framework.git", subdir = "aptos-framework", rev = "mainnet" }
+AptosStdlib = { local = "../../../aptos-move/framework/aptos-stdlib" }
+AptosFramework = { local = "../../../aptos-move/framework/aptos-framework" }
+
+# AptosStdlib = { git = "https://github.com/aptos-labs/aptos-framework.git", subdir = "aptos-stdlib", rev = "mainnet" }
```

Comment:
Why is this necessary?

Reply:
this stopped working recently:

```
thread 'main' panicked at crates/transaction-generator-lib/src/publishing/prebuild_packages.rs:119:58:
building package must succeed: Unable to resolve packages for package 'Aggregator examples'

Caused by:
    0: While resolving dependency 'AptosFramework' in package 'Aggregator examples'
    1: Failed to fetch to latest Git state for package 'AptosFramework', to skip set --skip-fetch-latest-git-deps | Exit status: exit status: 1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

branches other than mainnet work. But using local, instead of fetching, where available, seems reasonable/better? (i.e. if you want to make a framework change and test in examples)

Reply:
command is `cargo run -p module-publish`, and if you try it on main - it will fail with the above

Reply:
We can override where to fetch the libraries from using build options (see an example here:
https://github.com/aptos-labs/aptos-core/blob/c4b2790e53fabc118c06a02bee7fd163ad7638d5/aptos-move/move-examples/tests/move_unit_tests.rs#L37).

This would be the preferred mechanism.

Reply:
updated , works now. thanks!